packages/controller/src/lib/notificationHandler.ts

```diff
@@ -4,24 +4,117 @@
  *  2021 foxriver76 <moritz.heusinger@gmail.com>
  */
 
-const fs = require('fs-extra');
-const { tools } = require('@iobroker/js-controller-common');
-const path = require('path');
+import fs from 'fs-extra';
+import { tools } from '@iobroker/js-controller-common';
+import path from 'path';
+import type { Client as StatesInRedisClient } from '@iobroker/db-states-redis';
+import type { Client as ObjectsInRedisClient } from '@iobroker/db-objects-redis';
+import type Winston from 'winston';
+
+export interface NotificationHandlerSettings {
+    host: string;
+    states: StatesInRedisClient;
+    objects: ObjectsInRedisClient;
+    log: Winston.Logger;
+    logPrefix: string;
+}
 
-class NotificationHandler {
-    /**
-     * Create a new instance of NotificationHandler
-     *
-     * @param {object} settings - settings like states/objects db
-     */
-    constructor(settings) {
+interface MultilingualObject {
+    en: string;
+    [otherLanguages: string]: string;
+}
```

Comment:
I think this should be like `StringOrTranslated` [here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/e1070c9c06fb32be5f1f8b1d8af8e9045c76a9e4/types/iobroker/objects.d.ts#L133-L139) but without the `string`.

We want to model that `en` is required and the other possible languages are optional, but no other languages, right?

Reply:
good reminder, we already have it

Reply:
Hmm - is a plain `string` a valid option, i.e. is `StringOrTranslated` valid? Or does it have to be just the object part?
Because before your latest change, `string` was not allowed. Now it is.

Reply:
Ah thanks, no of course not. Now it should be correct.